moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -127,9 +126,14 @@ const useMakeFilterState = (filterQuery) =>
  * @return {JSX.Element}
  * @constructor
  */
-const ProjectsListViewTable = ({ query, searchTerm }) => {
+const ProjectsListViewTable = ({ query }) => {
   const classes = useStyles();
 
+  // create URLSearchParams from url
+  const navSearchTerm = useLocation()?.state?.searchTerm;
+  const filterQuery = useFilterQuery(useLocation().search);
```

Comment:
The multiple `useLocation` calls will be addressed in the advanced search PR. Didn't want to touch `filterQuery` for now.